sdk/ai/Azure.AI.Projects/tests/Samples/Extensions/Sample_AIInference.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
There's already a sample which basically already does this: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/ai/Azure.AI.Projects/tests/Samples/Inference/Sample_ChatCompletions.cs

Should that sample be updated? Or removed in favor of this one?

Reply:
Good catch. Thank you!